### ISSUE TITLE
[BUG] Content-Type doesn't match Reply body

### ISSUE THREAD

**Issue by ngosang** [immich-app/immich#5615]
### The bug

Immich is working well but I'm seeing a lot of traces like this in the log:
```
[Nest] 409  - 12/10/2023, 5:01:07 PM   ERROR [AssetService] Cannot create read stream for asset ec90291d-6d88-44ed-b3a4-ffedc27469f8

getAssetThumbnail

[Nest] 409  - 12/10/2023, 5:01:07 PM    WARN [ExpressAdapter] Content-Type doesn't match Reply body, you might need a custom ExceptionFilter for non-JSON responses
```

### The OS that Immich Server is running on

Ubuntu

### Version of Immich Server

v1.90.2

### Version of Immich Mobile App

v1.90.2

### Platform with the issue

- [X] Server
- [ ] Web
- [ ] Mobile

### Your docker-compose.yml content

```YAML
.
```

### Your .env content

```Shell
.
```

### Reproduction steps

```bash
.
```

**Comment by ForgetfulMagnet** [comment]
Bump. Seeing the same on my setup. Seems to have been introduced with 1.90.2.

**Comment by alextran1502** [comment]
Fixed in #5595 - you guys can ignore this error

**Comment by Zhan85** [comment]
This does not appear to be fixed.  Received this error just today.

**Comment by cybericius** [comment]
#metoo

**Comment by cybericius** [comment]
> Fixed in #5595 - you guys can ignore this error
Does this mean that the warning is still there, but we can ignore it, or that it doesn't suppose to appear?

**Comment by Tom-GitLab** [comment]
Error is still present in v1.97.0

**Comment by e-patrick** [comment]
Have the same message in the logs in v1.95.1. It appears always when I click on "Albums" in the left navigation.

**Comment by TobiZog** [comment]
I got this error when the browser progress bar hangs here:

![image](https://github.com/immich-app/immich/assets/67568441/89434f30-132f-41fd-8eb5-c33951eb2894)

**Comment by honulii** [comment]
I am also still receiving this error. It happens when I try to change location for many photos at once (somewhere over 100 photos)

**Comment by tysnxu** [comment]
This bug is still present, and the response body returns an in-complete JSON string for some reason. (ver 1.101.0)

It happened when I clicked on the home page. And it only happened for photos in the most recent week. Scrolling past the first week has no issues.

API: 
`/api/timeline/bucket?isArchived=false&size=MONTH&timeBucket=2024-04-01T00%3A00%3A00.000Z&withPartners=true&withStacked=true`

**Comment by babaz8** [comment]
Having this error too I don't know what is.

**Comment by maltokyo** [comment]
Dont know if this is a duplicate (as video playback is not mentioned here in this one), but I have created another issue with the same log output, as this issue seems stale, and I provided all relevant info.  New issue: https://github.com/immich-app/immich/issues/9351

**Comment by bo0tzz** [comment]
No need to create a new issue, just request this one to be reopened.

**Comment by maltokyo** [comment]
I am not sure if it is the same issue though, as I get that error only on video playback which is not mentioned here.

**Comment by maltokyo** [comment]
Anyway, if we can solve it here (with the info in that other issue) that is great too.

**Comment by maltokyo** [comment]
error comes from here: https://github.com/nestjs/nest/blob/6dc96d396b80d1b6e383948a8d297504e61aa671/packages/platform-express/adapters/express-adapter.ts#L111

**Comment by zigarn** [comment]
In my nginx access logs I can correlate this log with 499 status code (custom nginx status code for "client closed request").
Related URLs generating this log:
 - `/api/asset/thumbnail/{id}?format=WEBP`
 - `/api/asset/file/{id}?isThumb=true&isWeb=false`
The warning seems logic as those URL are returning binary content of images.

**Comment by jrasm91** [comment]
This was fixed (again) in #10013.
